lib/node_modules/@stdlib/blas/base/dtrmv/test/fixtures/column_major_l_t_nu.json

```diff
@@ -1,16 +1,21 @@
 {
   "order": "column-major",
+  "uplo": "lower",
   "trans": "transpose",
   "diag": "non-unit",
-  "uplo": "lower",
-  "strideX": 1,
-  "offsetA": 0,
-  "offsetX": 0,
+  "N": 3,
+  "A": [ 1.0, 2.0, 3.0, 0.0, 4.0, 5.0, 0.0, 0.0, 6.0 ],
+  "A_mat": [
+    [ 1.0, 2.0, 3.0 ],
```

Comment:
@ShabiShett07 I suggest revisiting these and ensuring that `A_mat` correctly reflects `uplo`.